ui/src/app/integrations/list-page/list-page.component.html

```diff
@@ -20,10 +20,13 @@
 </ng-template>
 </syndesis-list-toolbar>
 
-<syndesis-loading [loading]="loading | async">
-  <syndesis-integrations-list [complete]="true"
-                              [integrations]="filteredIntegrations | async"></syndesis-integrations-list>
-</syndesis-loading>
+<!-- Support importing by DND right into the integraiton list area. Very google drive like -->
+<div ng2FileDrop style="height:100%" [uploader]="uploader">
```

Comment:
There is no need to inject styles right into the DOM element. We already have `list-page.component.scss` for that.

Reply:
fixed